x-pack/plugins/fleet/server/services/security/uninstall_token_service/index.test.ts

```diff
@@ -34,13 +39,15 @@ describe('UninstallTokenService', () => {
             policy_id: 'test-policy-id',
             token: 'test-token',
           },
+          created_at: 'yesterday',
         }
       : {
           id: 'test-so-id',
           attributes: {
             policy_id: 'test-policy-id',
             token_plain: 'test-token-plain',
           },
+          created_at: 'yesterday',
```

Comment:
Is this valid?

Reply:
do you mean `yesterday`? it is just a dummy value, because `created_at` is a string timestamp